packages/lumos/clean.mjs

```diff
@@ -0,0 +1,59 @@
+import fs from "node:fs";
+import path from "node:path";
+import url from "node:url";
+
+const __dirname = path.dirname(url.fileURLToPath(import.meta.url));
+
+const whitelist = [
+  "node_modules",
+  "src",
+  ".gitignore",
+  ".npmignore",
+  "CHANGELOG.md",
+  "clean.mjs",
+  "package.json",
+  "README.md",
+  "tsconfig.json",
+  "typedoc.json",
+];
+
+function removeFilesExceptWhitelist(dir) {
```

Comment:
What directories other than `dist` or `build` are expected to be cleaned up by this script? is it convenient to just add a script in `package.json`, like `clean: rimraf ./build`

Reply:
The new build script now outputs all built files in the `packages/lumos` directory, making it more convenient for developers to import modules without a `/lib` or `/dist` when the `moduleRelution` is set to the default `node10` or `node`. Developers can import directly like this:

```js
import { secp256k1Blake160 } from '@ckb-lumos/lumos/common-scripts'
import { Uint8, Uint16, Uint32 } from '@ckb-lumos/lumos/codec'

// So developers needn't import modules like the following way
import { secp256k1Blake160 } from '@ckb-lumos/lumos/lib/commons-scripts'
```

This is more natural, and it's smoother to upgrade to the ESM in the future